### PR TITLE
Add CVE-2019-12399

### PR DESCRIPTION
**Pull request by rhauch**

### CODE REVIEW

File: cve-list.html [hunk 1]
```diff
@@ -8,6 +8,40 @@ <h1>Apache Kafka Security Vulnerabilities</h1>
 
 This page lists all security vulnerabilities fixed in released versions of Apache Kafka.
 
+<h2><a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12399">CVE-2019-12399</a>
+Apache Kafka Connect REST API may expose plaintext secrets in tasks endpoint</h2>
+
+<p>When Connect workers in Apache Kafka 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.2.0, 2.2.1, or 2.3.0 are
```
Comment (cmccabe) [inline cve-list.html:14]: It would be nice to split this into more than one sentence, since it's quite long

**Comment by cmccabe** [comment]
LGTM after fixing minor style nit

File: cve-list.html (hunk 1 above)
Comment (rhauch) [reply cve-list.html:14]: It is long, but the vulnerability only can occur when all of those criteria are met. Not sure how to break up the sentences without making it seem less restricted.

Comment (rhauch) [reply cve-list.html:14]: Also, for better or worse, it's the same text that's in the CVE.

Comment (cmccabe) [reply cve-list.html:14]: It could be broken up to be more readable.  If it's duplicating the CVE, though, then let's keep it as-is.

**Pull request merged**